server/models/ParticipantsModel/index.js

```diff
@@ -35,25 +55,58 @@ const allParticipantsQuery = (user, queryParams) => {
         complete: {
           $in: [subject, completed],
         },
+        Active: 1,
       },
     },
   ]
 
-  if (searchSubjects?.length)
-    query.push({
-      $match: {
-        $or: [
-          {
-            subject: {
-              $in: searchSubjects,
+  if (searchSubjects?.length) {
+    if (studies?.length) {
+      query.push({
+        $match: {
+          $or: [
+            {
+              study: {
+                $in: studies,
+              },
+            },
+            {
+              subject: {
+                $in: searchSubjects,
+              },
             },
+          ],
+        },
+      })
+    } else {
+      query.push({
+        $match: {
+          $or: [
+            {
+              subject: {
+                $in: searchSubjects,
+              },
+            },
+          ],
+        },
+      })
+    }
+  }
+
+  if (status) {
+    query.splice(1, 0, {
+      $project: {
+        subjects: {
+          $filter: {
+            input: '$subjects',
+            as: participant,
+            cond: { $eq: ['$$participant.Active', +status] },
```

Comment:
What does the `+` do in `+status`? Looks like it attempts to convert to a number https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus. If that's the intended behavior can we use `Number` for clarity?

Reply:
I second Jack's suggestion on `Number(status)` vs. `+status`, functionally equivalent but is a little more obvious about the type change

Reply:
yeah, thanks for the suggestion. To implement this change it would require updating the other places in the codebase that use operator to convert the values, and that's out of scope. Perhaps I can capture this and make these updates